rpcs3/Input/dualsense_pad_handler.h

```diff
@@ -0,0 +1,106 @@
+﻿#pragma once
+
+#include "Emu/Io/PadHandler.h"
+#include "Utilities/CRC.h"
+#include "hidapi.h"
+
+class dualsense_pad_handler final : public PadHandlerBase
+{
+	enum DualSenseKeyCodes
+	{
+		Triangle = 0,
+		Circle,
+		Cross,
+		Square,
+		Left,
+		Right,
+		Up,
+		Down,
+		R1,
+		//	R2But,
```

Comment:
R2But and L2But references can be removed here because they do not make much sense for dualsense.
They only exist in the DS4 handler, because the DS4 reports an additional (rather useless) "pressed" value next to the normal trigger value.